examples/jsm/controls/OrbitControls.js

```diff
@@ -614,12 +614,21 @@ class OrbitControls extends EventDispatcher {
 
 		function updateMouseParameters( event ) {
 
-			if ( ! scope.zoomToCursor ) {
+			if ( !scope.zoomToCursor ) {
```

Comment:
Do you mind undoing this change?

Reply:
the !zoomtocursor check is the origin code in r155, so I don’t think I can remove it
but I will undo the former one tomorrow

Reply:
I just mean the code style^^.